go/vt/wrangler/reparent.go

```diff
@@ -413,6 +436,89 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 	return nil
 }
 
+// maxReplPosSearch is a struct helping to search for a tablet with the largest replication
+// position querying status from all tablets in parallel.
+type maxReplPosSearch struct {
+	wrangler         *Wrangler
+	ctx              context.Context
+	waitSlaveTimeout time.Duration
+	waitGroup        sync.WaitGroup
+	maxPosLock       sync.Mutex
+	maxPos           replication.Position
+	maxPosTablet     *topodatapb.Tablet
+}
+
+func (maxPosSearch *maxReplPosSearch) processTablet(tablet *topodatapb.Tablet) {
+	defer maxPosSearch.waitGroup.Done()
+	maxPosSearch.wrangler.logger.Infof("getting replication position from %v", topoproto.TabletAliasString(tablet.Alias))
+
+	slaveStatusCtx, cancelSlaveStatus := context.WithTimeout(maxPosSearch.ctx, maxPosSearch.waitSlaveTimeout)
```

Comment:
I'd say, it's questionable. The second variable is a pointer to a function and the general rule is that function names should have verb as the main part. And indeed at the call site cancelSlaveStatus() looks much more natural than slaveStatusCancel().
So I think I'd prefer to keep it as it is unless you insist.